README.md

```diff
@@ -21,11 +21,71 @@ and reusable workflows.
+## Automatic Release Management
 
-To release a new version of the actions and workflows herein, simply follow the process to [create a release through GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).
-Taking the time to write a short summary of the changes to this version in the release description textbox would be nice for future users :wink:.
+### Introduction
 
-To keep the usage simple, we keep the version number to two digits: major (features) and minor (bug fixes, adjustments, etc). For example: "v1.1".
+This section outlines the release management process established for the *Foundation GitHub Actions* project.
+The release management approach ensures accurate and consistent versioning across all actions and workflows within
+the repository.
 
-Once the release is created, update the version used within the repository where needed. See the [GitHub examples using versioned actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions).
+### Release Philosophy
+
+#### Workflow Initiation
+To ensure accuracy throughout the release cycle and to avoid unnecessary version bumps,
+releases are initiated manually.
+
+#### Versioning
+A unified versioning system is implemented for all actions, ensuring uniformity across the entire project.
+This approach uses Lerna, which automatically adjusts the version of all actions and workflows in the monorepo.
+As a result, all the previous are updated to the same version and are released simultaneously.
+
+#### Release Condition
+Only PRs labeled with `auto-release` activate the releasing workflow. This label is automatically added in the
+`create-release-pr` workflow, which is triggered manually.
+
+#### Actions Versioning & Changelog
+With Lerna:
+- Versions are automatically adjusted based on established commit conventions.
+- A `CHANGELOG.md` is generated for every updated sub-package, detailing its changes.
+- `lerna version` is the command responsible for versioning and changelog generation.
+  For more details on the Lerna version command, visit [Lerna version command](https://github.com/lerna/lerna/tree/main/libs/commands/version#readme)
+
+#### Pull Request for New Release
+When a release is initiated, a Pull Request is created with the changelog and updated version. After this PR is
+approved and merged, the associated workflow will generate a release note and create a release with the proper tag.
+The unified versioning ensures a single release note and tag for the entire monorepo.
+
+#### Commit Versioning
+[Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) drive the version updates as follows:
+
+- A `fix` commit results in a patch bump (e.g., 1.0.0 to 1.0.1).
+- A `feat` commit results in a minor bump (e.g., 1.0.0 to 1.1.0).
+- The presence of a `BREAKING CHANGE:` in the commit footer leads to a major bump (e.g., 1.0.0 to 2.0.0).
```

Comment:
Just checking, will the release version have 3 digits for real?

Reply:
With Lerna and the way it works with conventional commit versioning, yes 😢